src/openms/source/CHEMISTRY/RibonucleotideDB.cpp

```diff
@@ -66,6 +87,209 @@ namespace OpenMS
     }
   }
 
+  // All valid JSON ribonucleotides must at minimum have elements defining name, short_name, reference_moiety, and formula
+  bool entryIsWellFormed_(const nlohmann::json::value_type& entry)
```

Comment:
not the behaviour I would expect.
If you return a bool, which is never `false`, why return something in the first place?! :)